custom/icds_reports/tasks.py

```diff
@@ -970,3 +972,8 @@ def create_mbt_for_month(state_id, month):
             icds_file, _ = IcdsFile.objects.get_or_create(blob_id='{}-{}-{}'.format(helper.base_tablename, state_id, month), data_type='mbt_{}'.format(helper.base_tablename))
             icds_file.store_file_in_blobdb(f, expired=THREE_MONTHS)
             icds_file.save()
+
+@task(queue='icds_dashboard_reports_queue')
```

Comment:
interesting choice of queue. was expecting the same agg queue. seems fine as long as there aren't any backups

Reply:
oh, yeah. that was not the queue i intended, i copy pasted from the mbt part, which also appears to be in the wrong queue. although i sort of like the idea that both of those don't trade off with the aggregation.